content/javascript/general-authoring/index.md

```diff
@@ -408,6 +408,25 @@ function yup(name, options, args) {
 }
 ```
 
+### Arrow Functions
+
+How to use arrow functions
+- Don't wrap a single argument with parenthesis.
+- When returning a function from an arrow function, create a block rather than returning on one line (helps readability).
```

Comment:
do we really feel so strongly against implicit return as to include this in the dev standards? personally I disagree, but if I'm in the vast minority that's fine